tests/otbs/argument_chain_indent.d.ref

```diff
@@ -0,0 +1,10 @@
+class C {
+    void f() {
+        if (true) {
+            f(map.map.map.map.map.map.map.map.map.map.map.map.map.map.map.map
+                    .map.map.map.map.map.map, __FILE__, __FILE__, __FILE__, __FILE__, __FILE__,
```

Comment:
shouldn't there be a new line after the `.map,`? Or at least wrap after the 3rd `__FILE__` and not after the 5th (because the other lines below also wrap around that column)

Reply:
Yeah, it is how line breaks are currently added. It was the main reason we wanted to have `--keep-line-breaks`, so such things can be adjusted manually because in the complicated code it is difficult to calculate the right way to add line breaks (if there is the right way).